index.html

```diff
@@ -63,6 +63,171 @@ <h1 class="intro">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <button class="works-button">See Project</button>
+        </div>
+        <div class="tonic d-flex">
+          <div class="works-image">
+            <img src="img/multipoststories.png" alt="Multi-post Stories">
+          </div>
+          <h2 class="works-header">Multi-post Stories</h2>
+          <ul class="header-sub d-flex">
+            <li class="main-header">CANOPY</li>
+            <li><img src="img/Counter.png" alt="Counter"></li>
+            <li>Back End Dev</li>
+            <li><img src="img/Counter.png" alt="Counter"></li>
+            <li>2015</li>
+          </ul>
+          <p class="works-text">A daily selection of privately persionalized reads ; no accounts or sign-ups required.</p>
+          <ul class="works-stack d-flex justify-content-between">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <button class="works-button">See Project</button>
+        </div>
+        <div class="tonic d-flex">
+          <div class="works-image">
+            <img src="img/tonic2.png" alt="Tonic Snapshoot">
+          </div>
+          <h2 class="works-header">Tonic</h2>
+          <ul class="header-sub d-flex">
+            <li class="main-header">CANOPY</li>
+            <li><img src="img/Counter.png" alt="Counter"></li>
+            <li>Back End Dev</li>
+            <li><img src="img/Counter.png" alt="Counter"></li>
+            <li>2015</li>
+          </ul>
+          <p class="works-text">A daily selection of privately persionalized reads ; no accounts or sign-ups required.</p>
+          <ul class="works-stack d-flex justify-content-between">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <button class="works-button">See Project</button>
+        </div>
+        <div class="tonic d-flex">
+          <div class="works-image">
+            <img src="img/multipoststories.png" alt="Multi-post Stories">
+          </div>
+          <h2 class="works-header">Multi-post Stories</h2>
+          <ul class="header-sub d-flex">
+            <li class="main-header">CANOPY</li>
+            <li><img src="img/Counter.png" alt="Counter"></li>
+            <li>Back End Dev</li>
+            <li><img src="img/Counter.png" alt="Counter"></li>
+            <li>2015</li>
+          </ul>
+          <p class="works-text">A daily selection of privately persionalized reads ; no accounts or sign-ups required.</p>
+          <ul class="works-stack d-flex justify-content-between">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <button class="works-button">See Project</button>
+        </div>
+      </section>
+      <section class="about d-flex">
+        <h1 class="about-header">About Myself</h1>
```

Comment:
- [ ] **[OPTIONAL]** Your project looks great, nice work! I noticed that you have used multiple `H1` tags on your page, and I was wondering if it would be possible to use a different heading tag for the subheadings, such as H2? This would help to maintain consistency and follow best practices. 🎯